Added short intro and setup instructions for docusaurus

Mikerniker opened the pull request:
Hello  @jhdalek55 / @Abhijay007
I tried adding a short intro to Uptane and setup instructions in the README.md based on the website and  @Abhijay007 's comments from PR [#84](https://github.com/uptane/uptane.github.io/pull/84). The original Jekyll instructions are still there (under the section I added), since I wasn't sure know if this should be deleted or not. Let me know if you need me to make any changes and if it meets your requirements for a Hacktoberfest PR. Thank you!

Abhijay007 commented on README.md:
```diff
@@ -1,4 +1,58 @@
+
+
+
+## Jekyll website
```
I think we don't need this info now as we have migrated the site to the doucusaurus

Abhijay007 commented on README.md:
```diff
@@ -84,6 +138,8 @@ Add assets, e.g. images or JavaScript, to [`assets`](assets).
 [`favicon.ico`](favicon.ico) should be served from the root of the project.
 Just replace the current one with the `favicon.ico` of your project.
 
```
till here

Mikerniker replied:
Ok, will remove the Jekyll section, should I wait to hear first from the other reviewers for other feedback before making the change or just go ahead and delete it? Thanks!

Abhijay007 replied:
ya it will be great if we wait for other's review first, so that you can do all the changes in one go

hexsecs commented on README.md:
```diff
@@ -84,6 +138,8 @@ Add assets, e.g. images or JavaScript, to [`assets`](assets).
 
+
+
 ## License
```
I believe @JustinCappos proposed changing the license. We should probably have a discussion regarding the licensing overall. For now we can keep text as is and fix in a pull request fixing the dedicated issue.

jhdalek55 reviewed (changes requested):
I made one suggested change in the intro. Though Uptane is not exclusive to protecting software updates over the air, it was designed for that purpose.

Mikerniker commented:
Hello @Abhijay007, @hexsecs, @jhdalek55
I've made additional changes to the pull request (i.e. incorporated the change to the Uptane description and deleted the Jekyll section). Could you please review it when you have time? Thank you!

hexsecs commented:
Fixes #85
